### PR TITLE
Update `README.md`

### PR DESCRIPTION
**Pull request by its-miroma**
Update `README.md`

- [x] Add right-aligned FO icon
  - [ ] Is there a distinct icon for light/dark mode?
- [x] Change "Launcher Comparison" table
  - Flip rows and columns, because items (Launchers) are usually placed as rows, plus if hypothetically there was a new one that would give a one-line diff
  - Remove "Can be used offline" column, because all launchers support that, and specified that above
  - Remove "Windows, macOS, Linux support" column, for the same reason
  - Remove "Unique feature" column, because I think FO, and especially this `README.md`, is not the right place to see which features launchers have
  - Place the table in a collapsible block
- [x] Add "Resource Packs" section, with links to download them.
- [x] Add "Contribute" section, linking to <https://download.fo/thanks>
- [x] Merge `CREDITS.md` into "Credits" section
- [x] Use [reference links][https://github.github.com/gfm/#reference-link] which help readability of the source file and make it more maintainable
- [ ] Add badges
- [ ] Move `INCLUDED-MODS.md` to the wiki?
- [ ] Add MultiMC auto update information
- [ ] Update `DEVELOPER-README.md`
  - [ ] Perhaps move to `CONTRIBUTING.md`?
  - [ ] Add link to "Contributing" section in `README.md`
  - [ ] Add link to `CODE_OF_CONDUCT.md` in `CONTRIBUTING.md`

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Code of Conduct #
+
+## DO ##
```
Comment (Madis0) [inline CODE_OF_CONDUCT.md:3]: What is the purpose of these changes (# on the right, * replaced with -) in most of the files when the rendered Markdown result is identical?

Comment (Kichura) [reply CODE_OF_CONDUCT.md:3]: Probably for raw result, however i can agree that this PR is essentially pointless.

Comment (its-miroma) [reply CODE_OF_CONDUCT.md:3]: Those changes were made automatically by my linter,
I apologize. I'll revert them

**Comment by its-miroma** [comment]
I hate my linter
